components/fxa-client/ios/FxAClient/FxAccountDevice.swift

```diff
@@ -142,7 +142,10 @@ public enum AccountEvent {
             return .deviceConnected(deviceName: msg.deviceConnectedName)
         }
         case .deviceDisconnected: do {
-            return .deviceDisconnected(isLocalDevice: msg.deviceDisconnectedIsLocal)
+            return .deviceDisconnected(
+                deviceId: msg.deviceDisconnectedData.deviceID,
```

Comment:
is `D` in `deviceID` correct here?

Reply:
[Yup](https://github.com/apple/swift-protobuf/issues/817)